src/user/user.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import * as bcrypt from 'bcrypt';
+
+@Injectable()
+export class UserService {
+  constructor(private prisma: PrismaService) {}
+
+  async getCurrentUser(userId: number) {
+    const user = await this.prisma.user.findUnique({ where: { id: userId } });
+    if (!user) {
+      throw new Error('User not found');
+    }
+    const { password, ...userWithoutPassword } = user;
+    return { user: userWithoutPassword };
+  }
+
+  async updateUser(userId: number, data: any) {
```

Comment:
```suggestion
  async updateUser(userId: number, data: <what is this?>) {
```
ko chơi `any` như này đâu,
em khai báo type tường minh ra nhé